tools/coverage.sh

```diff
@@ -1,3 +1,6 @@
+#!/bin/sh
```

Comment:
Please switch to bash.

https://google.github.io/styleguide/shell.xml?showone=Which_Shell_to_Use#Which_Shell_to_Use